Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Multiplayer Samples Co-op Changelog
 
+## [0.2.0] - 2021-05-18
+
+v0.2.0 is an Early Access release for Multiplayer Samples Co-op.
+
+It requires and supports Unity v2020.3 and later and Unity MLAPI v0.1.0. For additional information on MLAPI, see the changelog and release notes for the Unity MLAPI package.
+
+### New features
+
```

Comment:
Shouldn't we have one here letting users know that we've introduced static scene NetworkObjects? This may be the biggest improvement on the MLAPI side of things.

Reply:
Will double check tickets for one on this! Thanks!

Reply:
Yup. Feel free to align on this one offline if needed.